### DISCUSSION TITLE
Q: 评论区的头像是方形的该怎么办 || Q: What should I do if the avatar in the comment area is square?

### DISCUSSION THREAD

**Discussion by Real-king-Ph** [walinejs/waline#2145] [Q&A]
### Describe the bug | 问题描述

![image](https://github.com/walinejs/waline/assets/72426701/d0d28dbe-29d9-47fb-88e2-696287cb2fe8)

### Website URL | 问题网站

https://blog.ohtoai.fun/2023/11/15/23Y-Q4-10-23-Lecture-6-Memory-Management/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

![image](https://github.com/walinejs/waline/assets/72426701/d0d28dbe-29d9-47fb-88e2-696287cb2fe8)

### Website URL | Problem website

https://blog.ohtoai.fun/2023/11/15/23Y-Q4-10-23-Lecture-6-Memory-Management/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
看起来是 JS 和 CSS 引用的 CDN 加载的版本不一致，导致有部分样式缺失了。CSS 的地址替换成 https://cdn.jsdelivr.net/npm/@waline/client@2/dist/waline.css 试试。

---
It seems that the versions loaded by the CDN referenced by JS and CSS are inconsistent, causing some styles to be missing. Replace the CSS address with https://cdn.jsdelivr.net/npm/@waline/client@2/dist/waline.css and try it.
